admin/console/working-on-ticket-articles.rst

```diff
@@ -0,0 +1,19 @@
+Working with ticket articles
+****************************
+
+.. note:: Please note that this is not a full command list, if you're missing commands, feel free to ask over at the `Community <https://community.zammad.org>`_.
+
+Change notes to communication articles for SLA calculation
+----------------------------------------------------------
```

Comment:
```suggestion
Count Public “Notes” toward SLAs
--------------------------------
```

Reply:
That heading is **a lot** better than my initial one. Thanks!